operations/patch.go

```diff
@@ -83,7 +83,7 @@ func Patch() cli.Command {
 				return err
 			}
 
-			diffData, err := loadGitData(ref.Branch, args...)
+			diffData, err := loadGitData(ref.Branch, c.Bool(committedFlagName), args...)
```

Comment:
we do a weird thing here where we don't call into the context except at the beginning of the action functions, because I have this fantasy that we'll take most of the logic out of the action functions and make things more testable. Could you make this argument-access call more like the others, here and in the subsequent calls?

Reply:
done.